### DISCUSSION TITLE
Encryption feature not enabled

### DISCUSSION THREAD

**Discussion by ptthon** [openzfs/zfs#13439] [Q&A]
zfs version is:
zfs-0.8.6-1
zfs-kmod-0.7.5-1ubuntu16.12

According to the following steps to create the encrypted volume,but  could not create volume
① To install the following commands to compile the installation ZFS
sudo tar -xvzf zfs-0.8.6.tar.gz
sudo cd zfs-0.8.6
sudo ./configure
sudo make
sudo make install
② truncate -s 100G /tmp/disk.img
     zpool create zfspv-pool `sudo losetup -f /tmp/disk.img --show`

③ zfs create -o encryption=on -o keyformat=passphrase -o keylocation=prompt zfspv-pool/enc
the error is : cannot create 'zfspv-pool/enc': Encryption feature not enabled.

④ I try to execute the following commands,but still cannot create encrypted volume
   zpool set feature@encryption=enabled

**Comment by mschilli87** [comment]
My guess is that `zfs-kmod-0.7.5-1ubuntu16.12` needs to be upgraded to match `zfs-0.8.6-1`: Your kernel module is too old to know of the encryption feature. Please refer to the Ubuntu help channels for further advice as this does not appear to be an issue with zfs itself and its maintainers won't be able to change anything about how Ubuntu packages/ships zfs.

**Reply by ptthon** [reply]
Thank you very much, I update the zfs-kmod, would be great
